2D/settings.py

```diff
@@ -68,3 +68,4 @@
 USE_UPSAMPLING = False
 USE_AUGMENTATION = True  # Use data augmentation during training
 USE_DROPOUT = False  # Use spatial dropout in model
+USE_PCONV = True   # If True, Partial Convolution based padding will be used. See https://arxiv.org/pdf/1811.11718.pdf
```

Comment:
Can you set the default to False?  Otherwise code looks good. Thanks.

Reply:
Good catch, forgot to change it to false after testing ;)